### ISSUE TITLE
English word in the French version of UMIL

### ISSUE THREAD

**Issue by fujinaga** [DDMAL/VIM#160]
Horbostel-Sachs box in the Instruments page, the French version says: "Unclassifed".
Shouldn't it be changed to **Non classifié** ?

<img width="241" alt="image" src="https://github.com/user-attachments/assets/cc37c94b-5fc3-4e54-82d2-481c033b0c43">

**Comment by JoyfulGen** [comment]
I think "non classifié" is good! Although perhaps the plural "Non classifiés" might be better, supposing there will be more than one unclassified instrument?

**Comment by fujinaga** [comment]
Yes, let’s change it to “Non classifiés”. Thank you, @JoyfulGen !
